File: Grazioso/Monkey.java
public class Monkey extends RescueAnimal {
    // Private attributes
    private String tailLength;
    private String height;
    private String bodyLength;
    private String species;

    // Constructor
    public Monkey(String name, String species, String tailLength, String height, String bodyLength,
                  String gender, String age, String weight, String acquisitionDate, String acquisitionCountry,
                  String trainingStatus, boolean reserved, String inServiceCountry, String animalType) {
        setName(name);
        setSpecies(species);
        setTailLength(tailLength);
        setHeight(height);
        setBodyLength(bodyLength);
        setGender(gender);
        setAge(age);
        setWeight(weight);
        setAcquisitionDate(acquisitionDate);
        setAcquisitionLocation(acquisitionCountry);
        setTrainingStatus(trainingStatus);
        setReserved(reserved);
        setInServiceCountry(inServiceCountry);
        setAnimalType(animalType);
    }

    // Mutator methods
    public void setTailLength(String tailLength) {
        this.tailLength = tailLength;
    }
    public void setHeight(String height) {
        this.height = height;
    }
    public void setBodyLength(String bodyLength) {
        this.bodyLength = bodyLength;
    }
    public void setSpecies(String species) {
        this.species = species;
    }

    // Accessor methods
    public String getTailLength() {
        return tailLength;
    }
    public String getHeight() {
        return height;
    }
    public String getBodyLength() {
        return bodyLength;
    }
    public String getSpecies() {
        return species;
    }
}

File: Grazioso - Enhancement 1 Python/Monkey.py
from RescueAnimal import RescueAnimal

# Monkey class extends RescueAnimal
class Monkey(RescueAnimal):
    
    # Default constructor
    def __init__(self, name, species, tailLength, height, 
               bodyLength, gender, age, weight, acquisitionDate, 
               acquisitionCountry, trainingStatus, reserved, 
               inServiceCountry, animalType):
        
        # Superclass constructor
        super().__init__()
        self.tailLength = tailLength
        self.height = height
        self.bodyLength = bodyLength
        self.species = species

        self.setName(name)
        self.setGender(gender)
        self.setAge(age)
        self.setWeight(weight)
        self.setAcquisitionDate(acquisitionDate)
        self.setAcquisitionCountry(acquisitionCountry)
        self.setTrainingStatus(trainingStatus)
        self.setReserved(reserved)
        self.setInServiceCountry(inServiceCountry)
        self.setAnimalType(animalType)
    
    #Getters
    def getTailLength(self):
        return self.tailLength
    
    def getHeight(self):
        return self.height
    
    def getBodyLength(self):
        return self.bodyLength
    
    def getSpecies(self):
        return self.species
    
    #Setters
    def setTailLength(self, tailLength):
        self.tailLength = tailLength
    
    def setHeight(self, height):
        self.height = height
    
    def setBodyLength(self, bodyLength):
        self.bodyLength = bodyLength
    
    def setSpecies(self, species):
        self.species = species

